docs/api/index.md

```diff
@@ -497,6 +497,26 @@ When you use `test` or `bench` in the top level of file, they are collected as p
   })
   ```
 
+### describe.skipIf
+
+- **Type:** `(condition: any) => void`
+
+  In some cases you might run suites multiple times with different environments, and some of the suites might be environment-specific. Instead of wrapping the suite with `if`, you can use `describe.skipIf` to skip the suite whenever the condition is truthy.
```

Comment:
```suggestion
  In some cases, you might run suites multiple times with different environments, and some of the suites might be environment-specific. Instead of wrapping the suite with `if`, you can use `describe.skipIf` to skip the suite whenever the condition is truthy.
```